backend/src/services/cars/cars-search.service.ts

```diff
@@ -3,13 +3,14 @@ import { prisma } from '@data/prisma-client';
 
 const carsSearchAutoria = async (
   complectationId: string,
-  page = '1',
+  page = '0',
+  countpage = 10,
 ): Promise<Partial<AutoriaRequestParams>> => {
   const result: Partial<AutoriaRequestParams> = {
     category_id: 1,
     power_name: 1,
```

Comment:
Is that correct?

Reply:
Yes, it is correct:
 category_id: 1 mean "cars"
 power_name: 1 mean "Power unit - hp"